File: tests/fixtures/open_redirect/php/unsafe_redirect.php
<?php
// Unsafe: $_GET['next'] flows directly into a `header("Location: ...")`
// line.  The PHP gated SinkGate for `header` activates on the
// `Location:` first-arg prefix and emits OPEN_REDIRECT in addition to
// the existing flat HEADER_INJECTION sink.
$next = $_GET['next'];
header("Location: " . $next);

File: tests/fixtures/open_redirect/go/unsafe_redirect.go
package handler

import "net/http"

// Unsafe: query arg flows directly into http.Redirect (3rd arg URL).
func Unsafe(w http.ResponseWriter, r *http.Request) {
	target := r.URL.Query().Get("next")
	http.Redirect(w, r, target, http.StatusFound)
}
